README.md

```diff
@@ -117,6 +118,22 @@ You can find a lot more about configuring Grist, setting up authentication,
 and running it on a public server in our
 [Self-Managed Grist](https://support.getgrist.com/self-managed/) handbook.
 
+## Available Docker images
+
+The default Docker image is `gristlabs/grist`. This contains all of
+the standard Grist functionality, as well as extra source-available
+code for enterprise customers taken from the the
+[grist-ee](https://github.com/gristlabs/grist-ee) repository. This
+extra code is not under a free or open source license. By default,
+however, the code from the `grist-ee` repository is completely inert and
+inactive. This code becomes active only when an administrator enables
+it from [the admininistrator panel](#the-admin-panel).
```

Comment:
At the point when we likely land this, the panel option won't be present, and the extra code will become active when an administrator sets an activation key in the environment.

Reply:
The anchor link is wrong too, so I amended this passage.